curious/commands/context.py

```diff
@@ -187,6 +187,12 @@ def _make_reraise_ctx(self, new_name: str) -> EventContext:
 
         :return: If it can be ran, and a list of conditions that failed.
         """
+        if getattr(cmd, "cmd_owner_bypass", False):
+            application = self.bot.application_info
+            if application is not None:
+                if self.author.id == application.owner.id:
```

Comment:
Use `self.message.author_id` instead. Faster, and won't explode out of nowhere if Discord sends terrible packets.